Mathlib/CategoryTheory/Limits/FunctorCategory/Basic.lean

```diff
@@ -437,13 +437,34 @@ the individual limits on objects. -/
 def limitIsoFlipCompLim [HasLimitsOfShape J C] (F : J ⥤ K ⥤ C) : limit F ≅ F.flip ⋙ lim :=
   NatIso.ofComponents (limitObjIsoLimitCompEvaluation F)
 
+/-- `limitIsoFlipCompLim` is natural in `F`. -/
```

Comment:
Docstrings must make sense with minimal context. Even if the convention is to call functors `F`, I’d suggest
```suggestion
/-- `limitIsoFlipCompLim` is natural with respect to diagrams. -/
```

Same for other docstrings.